plugins/epos/plugin.py

```diff
@@ -265,18 +265,40 @@ def rda_a1_02m(self):
         """
         # 2 - Look for the metadata terms in HTML in order to know if they can be accessed manually
 
-        return (0, "")
+        msg = "The link to the manual obtention  of the metadata is " + str(
+            self.manual_metadata
```

Comment:
where is this variable defined?

Reply:
It dissapeared while separating the PRs. Now it is definned after getting the metadata